lua/astrocommunity/colorscheme/oldworld-nvim/init.lua

```diff
@@ -0,0 +1,3 @@
+return {
+  "dgox16/oldworld.nvim",
```

Comment:
```suggestion
  "dgox16/oldworld.nvim",
  opts = {}, 
```

Reply:
looking at the docs `setup` isn't required